lib/idx/idxState/v1/generateIdxAction.ts

```diff
@@ -36,13 +37,18 @@ const generateDirectFetch = function generateDirectFetch(authClient: OktaAuthIdx
     });
 
     try {
-      const response = await httpRequest(authClient, {
+      const options: RequestOptions = {
         url: target,
         method: actionDefinition.method,
         headers,
         args: body,
         withCredentials: toPersist?.withCredentials ?? true
-      });
+      };
+      const isPolling = target.endsWith('/poll');
```

Comment:
I'm not sure we can guarantee all IDX polling flows utilize paths that end in `/poll`

Reply:
changed to
```
const isPolling = actionDefinition.name === 'poll' || actionDefinition.name?.endsWith('-poll');
```